src/get_ensemble_output.R

```diff
@@ -0,0 +1,30 @@
+library(dplyr)
+library(ggplot2)
+library(hubUtils)
+library(hubVis)
+library(hubEnsembles)
+
+model_names = c("epiENGAGE-GBQR", "epiENGAGE-Copycat", "epiENGAGE-INFLAenza", "epiforecasts-dyngam")
+reference_date = "2025-03-15"
+
+for(i in 1:length(model_names)){
+  assign("tmp", read.csv(paste("https://raw.githubusercontent.com/reichlab/flu-metrocast/main/model-output/", model_names[i], "/", reference_date, "-", model_names[i], ".csv", sep="")))
+  assign(paste("output",i, sep=""), get("tmp") %>% mutate(model_id = model_names[i]))
+}
+
+forecast_output <- output1 %>% 
+  bind_rows(output2) %>%
+  bind_rows(output3) %>%
+  bind_rows(output4) 
```

Comment:
I suggest using code from hubData to extract all of the data at once, without having to loop over files. See the vignette here for more info: https://hubverse-org.github.io/hubData/articles/connect_hub.html